ext/semian/semian.c

```diff
@@ -489,7 +491,7 @@ void Init_semian()
   eInternal = rb_const_get(cSemian, rb_intern("InternalError"));
 
   rb_define_alloc_func(cResource, semian_resource_alloc);
-  rb_define_method(cResource, "initialize", semian_resource_initialize, 4);
+  rb_define_method(cResource, "_initialize", semian_resource_initialize, 4);
```

Comment:
See @fw42's comment. That allow me to have override the C constructor with Ruby code.

It's a fairly common practice for C extensions.